types/Page.php

```diff
@@ -162,6 +162,10 @@ public function sort(QueryBuilder $QB, $tablealias, $colname, $order) {
      */
     public function rawValue($value) {
         if($this->config['usetitles']) {
+            if (cleanID($value) == $value) {
+                // This is already a raw value
+                return $value;
+            }
```

Comment:
I don't like this. I think we should go the hard way and fix this properly in entry.php (or where ever).

I wish strings where real objects and we could enhance them with a flag to note what kind of value it is. Hmm... maybe we should do that? *sigh* those complex types introduced so much hardship, I wonder if they were worth it...